articles/Monerium/monerium.md

```diff
@@ -0,0 +1,155 @@
+
+Now without going into the details of what step 1-3 under the rules of risk weighting exactly means and how it is calculated, we will boil down the variation of possible and relevant secure and low-risk liquid assets to the table below for the sake of simplicity:
+
+> Debt securities issued or guaranteed by central governments, issued by central banks, international organisations, multilateral development banks or Member States' regional government or local authorities. For example government bonds.
+> 
+> With a risk capital charge of [0% which equals a AAA to AA- rating](https://www.bis.org/basel_framework/chapter/MAR/20.htm#:~:text=The%20specific%20risk%20capital%20requirements%20for%20%E2%80%9Cgovernment%E2%80%9D%20and%20%E2%80%9Cother%E2%80%9D%20categories%20will%20be%20as%20follows).
+
+> The same type of debt securities as before but with a risk capital charge of 0.25% to 1.6% which equals A+ to BBB- rating.  
+or debt securities issued by AAA to A-rated financial institutions  
+or debt securities issued by AAA to A-rated corporates
+>
+> [Reference for the rating scale with regard to the credit quality step](https://www.eiopa.europa.eu/sites/default/files/publications/amended_draft_mapping_report_-_spread_research_abridged.pdf)
+
+> A third option is [UCITS](https://www.investopedia.com/terms/u/ucits.asp) funds, which invest in the very same assets as specified above. This could be a fund vehicle provided by asset managers (like BlackRock) that can be used by an EMI to "outsource" the operations behind managing the reserves. 
+
+### Risk Summary E-money:
+
+It is important to note that there are residual risks associated with e-money, such as defaulting governments or banks. However, considering that an EMI is likely banking with systemic relevant banks and investing in AAA-rated gov. bonds the remaining risks can be practically disregarded. As a matter of fact, e-money is considerably less risky than the systemic relevant banks which operate in the EU. Firstly, unlike banks that operate under the fractional reserve system, e-money is over-collateralized with 2% of the outstanding issued e-money, as well as a capital contribution of 350k. Secondly, most of the assets are invested into high-quality liquid assets and only the necessary amount is kept at a bank to meet regular withdrawals (as part of a prudent practice), whereby the reserves are also segregated from the EMI. Thirdly, gov. bonds are secured through the ability of the government to tax its citizens, while banks do not have such an option. And lastly, you have diversification benefits by nature of investing in multiple different gov. bonds. All of these factors combined make the case for money that provides a more secure money scheme than demand deposits.
```

Comment:
same comment here on over-collateralization, it seems it's a full reserve requirement + additional capital requirements?

Reply:
Okay I think I understand your first remark more now

Reply:
See the comment above. Monerium EMI safeguards 100% of its customer funds in addition to providing at least 2% of its own capital to meet its liabilities.